### PR TITLE
feat(Buttons): update buttons to the new design - T10725

### PR DESCRIPTION
**Pull request by NermeenMoustafa**

### CODE REVIEW

**Comment by wagdy-git** [comment]
reviewed

File: packages/ui/src/core/button/button.component.tsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ const Button = intrinsicComponent<ButtonProps, HTMLButtonElement>(
           <Badge
             inline
             size={14}
-            fontSize={10}
+            fontSize={ButtonSize.Md}
```
Comment (amrelbialy) [inline packages/ui/src/core/button/button.component.tsx:45]: why change fontSize to string and i don't see badge component supporting this

File: packages/ui/src/theme/roots/palette/entity/default-palette.ts [hunk 2]
```diff
@@ -51,38 +51,44 @@ export const lightPalette = {
...
   [Color.BackgroundSecondary]: 'rgba(255,255,255,1)',
 
-  [Color.IconsPrimary]: 'rgba(93,109,126,1)',
+  [Color.IconsPrimary]: '#6879EB',
   [Color.IconsPrimaryOpacity_0_6]: 'rgba(93,109,126,0.600)',
   [Color.IconsSecondary]: '#959DA8',
   [Color.IconsPlaceholder]: '#DEE6ED',
+  [Color.IconButtonBasicHoverColor]: ' #4f6276',
+  [Color.IconButtonBasicHoverBackground]: ' #f9fbfc',
+  [Color.IconButtonBasicActiveColor]: ' #37414b',
```
Comment (amrelbialy) [inline packages/ui/src/theme/roots/palette/entity/default-palette.ts:71]: Actually we don't need this new colors, we should use the same color used in figma. If you click an item in figma you will see the name of the color used and it's value. If you find the same name in UI kit but different value from figma you should update it in UI kit to the latest value in figma

So please check all colors that you added and check again with figma

File: packages/ui/src/theme/roots/palette/entity/default-palette.ts [hunk 3]
```diff
@@ -51,38 +51,44 @@ export const lightPalette = {
...
+  [Color.AccentPrimaryDisabled]: 'rgba(104, 121, 235, 0.12)',
 
   [Color.BackgroundPrimary]: 'rgba(248,250,251,1)',
   [Color.BackgroundPrimaryHover]: 'rgba(239,243,246,1)',
   [Color.BackgroundPrimaryActive]: 'rgba(93,109,126,1)',
   [Color.BackgroundPrimary_0_5_Opacity]: 'rgba(255,255,255,0.500)',
   [Color.BackgroundSecondary]: 'rgba(255,255,255,1)',
 
-  [Color.IconsPrimary]: 'rgba(93,109,126,1)',
+  [Color.IconsPrimary]: '#6879EB',
```
Comment (amrelbialy) [inline packages/ui/src/theme/roots/palette/entity/default-palette.ts:65]: Icon primary shouldn't be the same as accent

**Review by amrelbialy — APPROVED** [review]

**Pull request merged**